Driver/Driver/Util/UserDefaultsManager.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  UserDefaultsManager.swift
+//  Driver
+//
+//  Created by Fatih Doğan on 17.05.2023.
+//
+
+import Foundation
+
+struct UserDefaultsManager {
+
+    private static let KEY_ACCESS_TOKEN = "auth_token"
+    private static let KEY_ACCESS_TOKEN_EXPIRE = "auth_token_expire"
+    private static let KEY_REFRESH_TOKEN = "refresh_token"
+    
+    static func saveAuthTokens(tokens: Token) {
+        let defaults = UserDefaults.standard
+        defaults.set(tokens.accessToken, forKey: UserDefaultsManager.KEY_ACCESS_TOKEN)
+        defaults.set(tokens.accessTokenExpiresAt, forKey: UserDefaultsManager.KEY_ACCESS_TOKEN_EXPIRE)
+        defaults.set(tokens.refreshToken, forKey: UserDefaultsManager.KEY_REFRESH_TOKEN)
+    }
+    
+    static func getAccessToken() -> TokenInfo {
+        let defaults = UserDefaults.standard
```

Comment:
you can use **UserDefaults.standard** as a parameter of this class. You don't need to call it for all methods scope.